source/opt/inline_pass.h

```diff
@@ -117,11 +134,31 @@ class InlinePass : public Pass {
                      ir::UptrVectorIterator<ir::Instruction> call_inst_itr,
                      ir::UptrVectorIterator<ir::BasicBlock> call_block_itr);
 
-  // Returns true if |inst| is a function call that can be inlined.
+  // Return true if |inst| is a function call that can be inlined.
   bool IsInlinableFunctionCall(const ir::Instruction* inst);
 
-  // Returns true if |func| is a function that can be inlined.
-  bool IsInlinableFunction(const ir::Function* func);
+  // Compute structured successors for function |func|.
+  // A block's structured successors are preceded by its
```

Comment:
Update: after reading the body of the function I think I understand now.  Except now the comment is weird:  For a given block B, is B's declared merge block a structured successor to B or not?   It think yes, but your sentence here implies otherwise.


Suggest something like this to replace this middle sentence:
```
The structured successors of a block B are the set of the blocks it branches to, together with its declared merge block if it has one.  When order matters, the merge block always appears first.
```